EIPS/eip-1271.md

```diff
@@ -41,19 +41,19 @@ pragma solidity ^0.5.0;
 contract ERC1271 {
 
   // bytes4(keccak256("isValidSignature(bytes,bytes)")
```

Comment:
Comment should be changed to `bytes32,bytes`

Reply:
LGTM otherwise!

Reply:
fixed